### ISSUE TITLE
Multiple different osm tags for a single operator?

### ISSUE THREAD

**Issue by k-yle** [osmlab/name-suggestion-index#6143]
Is there a way to specify multiple different osm tags for a single operator? For example, companies like _Live Nation_ and _ASM Global_ would require 6 entries in NSI, since they operate 6 different OSM features:
* `amenity=theatre`
* `amenity=conference_centre`
* `leisure=stadium`
* `amenity=events_venue`
*  `amenity=cafe`
* `shop=ticket`

Is there a smarter way than adding heaps of entries for the same operator?

**Comment by bhousel** [comment]
> Is there a smarter way than adding heaps of entries for the same operator?

Yeah this is what we do currently, and there isn't really a better way right now.  The power features also tend to have a lot of redundancy (like substations, lines, generators, etc).

It is also possible to use template items to copy all or some the items from one category into another.
For example the `power/minor_line` category is just a copy of everything in `power/line` + a few entries.
https://github.com/osmlab/name-suggestion-index/blob/main/data/operators/power/minor_line.json#L8-L11
